UnityProject/Assets/Scripts/Objects/VisibleBehaviour.cs

```diff
@@ -109,7 +134,12 @@ private void UpdateState(bool _aliveState)
 		}
 	}
 
-	private bool canBeDisabled(MonoBehaviour script)
+	private bool CanBeDisabled(SpriteRenderer script)
+	{

```

Comment:
I know its a small thing but the parameter here should not be called script. Call it something related to a renderer